src/main/java/org/datadog/jmxfetch/AppConfig.java

```diff
@@ -96,6 +96,13 @@ public class AppConfig {
     @Builder.Default
     private boolean logFormatRfc3339 = false;
 
+    @Parameter(
+        names = {"--log_thread_name"},
+        description = "Logs the thread name with each message",
+        required = false)
+    @Builder.Default
+    private boolean logThreadName = false;
```

Comment:
There is nothing wrong with enabling it by default or actually just always doing it and removing this flag. Only reason I didn't do it as was following the same pattern used by the milliseconds flag. I guess if someone is parsing the logs already then this would mean their parser breaks.

Reply:
I think milliseconds are optional, to keep the default format aligned with the agent, which doesn't use millisecond timestamps.

> same pattern used by the milliseconds flag

It's not quite the same pattern though: milliseconds option uses a java property as a toggle, not a command-line flag. This difference is somewhat of consequence, since users can set the properties via `java_options`, but IIRC there is no way to add arbitrary app-level flags (all existing flags are hardcoded in the `jmxfetch.go`).